ci/ansible/roles/pulp-certs/tasks/main.yml

```diff
@@ -20,13 +20,13 @@
 - name: Configure openssl subjectAltName
   lineinfile: dest=/etc/pki/tls/openssl.cnf
               insertafter="^\[ usr_cert \]"
-              line="subjectAltName=DNS:{{ansible_fqdn}}"
+              line="subjectAltName=DNS:{{ ansible_fqdn }},DNS:{{ ansible_nodename }},DNS:{{ ansible_hostname }}"
```

Comment:
This looks reasonable to me.

Out of curiosity, do you know what the difference might be between the nodename and the fqdn? In my brief search of the Ansible documentation I wasn't able to find out the exact difference.